client/shared/src/components/ResultContainer.tsx

```diff
@@ -150,6 +160,15 @@ export const ResultContainer: React.FunctionComponent<Props> = ({
                         )}
                     </button>
                 )}
+                {matchCountLabel && isRedesignEnabled && formattedRepositoryStarCount && (
```

Comment:
```suggestion
                {matchCountLabel && formattedRepositoryStarCount && (
```
We can assume `isRedesignEnabled` is not needed anymore. If you want you can completely remove it from this class.